try:
    print("hello world")
except Exception as e:
    print(e)